src/app/pages/list/multiselect-list/multiselect-list.component.html

```diff
@@ -0,0 +1,40 @@
+<mat-toolbar color="primary">
+    <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title">Multiselect List</div>
+    <pxb-spacer></pxb-spacer>
+    <button mat-icon-button (click)="onAddItem()"><mat-icon>add</mat-icon></button>
+</mat-toolbar>
+<mat-nav-list>
+    <pxb-info-list-item *ngFor="let item of data">
```

Comment:
Is our pxb-info-list-item automatically doing the hover/pointer/ripple behavior? If so, we should log an issue to fix that.

Reply:
Is this because it's in a mat-nav-list? Should we switch to mat-list if we want that off?

Reply:
It was the mat-nav-list causing that behavior. Switching it to mat-list does the trick.